example/sctp.go

```diff
@@ -29,20 +29,26 @@ func main() {
 	var server = flag.Bool("server", false, "")
 	var ip = flag.String("ip", "0.0.0.0", "")
 	var port = flag.Int("port", 0, "")
+	var lport = flag.Int("lport", 0, "")
+
 	flag.Parse()
 
 	ips := []net.IPAddr{}
 
 	for _, i := range strings.Split(*ip, ",") {
 		if a, err := net.ResolveIPAddr("ip", i); err == nil {
+			log.Printf("Resolvied address '%s' to %s", i, a)
```

Comment:
typo

Reply:
sure, will fix the typo.
I don't think we should remove https://github.com/ishidawataru/sctp/blob/master/sctp_linux.go#L140-L147
Listen and Dial can make better guess on IPv4 vs IPv6 since they also have network parameter which can specify protocol explicitly.  ToRawSockAddrBuf does not have any extra hints on the protocol and has to fall back to IPv4 zero, it's better then nothing, but not as precise as Dial & Listen guesses.